src/TimeSeries/TimeSeries.xml

```diff
@@ -0,0 +1,121 @@
+            <category>X-axis</category>
+            <description>Label for the x-axis (horizontal)</description>
+        </property>
+        <property key="xAxisFormat" type="string" required="false">
+            <caption>X-axis data format</caption>
+            <category>X-axis</category>
+            <description>Formats of date on x-axis (use NVD3 date formatting)</description>
+        </property>
+        <property key="yAxisLabel" type="string" required="false">
+            <caption>Y-axis label</caption>
+            <category>Y-axis</category>
+            <description>Label for the y-axis (vertical)</description>
+        </property>
+        <property key="yAxisFormat" type="string" required="false">
+            <caption>Y-axis data format</caption>
+            <category>Y-axis</category>
+            <description>Format of data on y-axis</description>
+        </property>        
+        <property key="seriesConfig" type="object" isList="true">
+            <caption>Series</caption>
+            <category>Series Configuration</category>
+            <description>Add a series and configure its properties</description>
+            <properties>
+                <property key="seriesEntity" type="entity" pathType="reference" isPath="optional">
+                    <caption>Data entity</caption>
+                    <category>Data source</category>
+                    <description>The entity containing series points</description>
+                </property>
+                <property key="seriesSource" type="enumeration" defaultValue="xpath">
+                    <caption>Source</caption>
+                    <category>Data source</category>
+                    <description>Type of source</description>
+                    <enumerationValues>
+                        <enumerationValue key="xpath">XPath</enumerationValue>
+                        <enumerationValue key="microflow">Microflow</enumerationValue>
+                    </enumerationValues> 
+                </property>
+                <property key="seriesXAttribute" type="attribute" entityProperty="seriesEntity">
+                    <caption>X-axis data</caption>
+                    <category>Data source</category>
+                    <description>Attribute containing dates for x-axis</description>
+                    <attributeTypes>
+                        <attributeType name="DateTime"/>
+                    </attributeTypes>
+                </property>
+               <property key="seriesYAttribute" type="attribute" entityProperty="seriesEntity">
+                    <caption>Y-axis data</caption>
+                    <category>Data source</category>
+                    <description>Attribute containing data for y-axis</description>
+                    <attributeTypes>
+                        <attributeType name="Decimal"/>
+                    </attributeTypes>
+                </property>
+                <property key="seriesKey" type="string">
```

Comment:
The key is always string, its actually a name of the series. This has been renamed to "name". Its used in the legend.